python2/httplib2/__init__.py

```diff
@@ -1740,9 +1756,106 @@ def clear_credentials(self):
         self.credentials.clear()
```

Comment:
Also clear cookies on `Http.close()`
```suggestion
        self.cookies.clear()
        self.credentials.clear()
```